src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -1091,126 +1492,132 @@ void func_80A137C0(EnGo* this, PlayState* play, f32 arg2, f32 arg3) {
     }
 }
 
-void func_80A139E4(EnGo* this) {
-    static Vec3f D_80A16698 = { 0.0f, 0.06f, 0.0f };
-    Vec3f sp54;
-    Vec3f sp48;
-    s16 sp46 = Rand_ZeroOne() * 360.0f * 182.0f;
-
-    Math_Vec3f_Copy(&sp54, &gZeroVec3f);
-    sp54.z = 28.0f;
-    Lib_Vec3f_TranslateAndRotateY(&this->actor.world.pos, sp46, &sp54, &sp48);
-    sp48.y = (Rand_ZeroOne() * 10.0f) + 4.0f;
-    sp48.y += this->actor.floorHeight;
-    func_80A10FD0(&this->unk_3F8[16], sp48, D_80A16698, gZeroVec3f, 0.01f, 0.002f, 16);
-}
-
-s32 func_80A13B1C(EnGo* this, PlayState* play) {
+/**
+ * Initialize the steam effect used for melting ice.
+ */
+void EnGo_MakeSteam(EnGo* this) {
+    static Vec3f accel = { 0.0f, 0.06f, 0.0f };
```

Comment:
```suggestion
    static Vec3f sAccel = { 0.0f, 0.06f, 0.0f };
```
We still try to use the `s` prefix even for in-function data